src/importlinter/adapters/user_options.py

```diff
@@ -51,3 +61,38 @@ def _clean_section_config(section_config: Dict[str, Any]) -> Dict[str, Any]:
             else:
                 section_dict[key] = value.strip().split("\n")
         return section_dict
+
+
+class TomlUserOptionReader(ports.UserOptionReader):
```

Comment:
For consistency, shouldn't this be `TomlFileUserOptionReader`?

Reply:
done